iohk-monitoring/src/Cardano/BM/Backend/Log.lhs

```diff
@@ -126,10 +124,11 @@ instance (ToJSON a, FromJSON a) => IsBackend Log a where
         let updateEnv :: K.LogEnv -> IO UTCTime -> K.LogEnv
             updateEnv le timer =
                 le { K._logEnvTimer = timer, K._logEnvHost = "hostname" }
-        cfoKey <- Config.getOptionOrDefault config (pack "cfokey") (pack "<unknown>")
+        era <- Config.getOptionOrDefault config (pack "era") (pack "Byron")
```

Comment:
a better name would probably be: "appversion"
(the version of the application that includes iohk-monitoring fw)

Reply:
done

Reply:
Done.